protocol-units/settlement/mcr/contracts/test/token/MOVETokenV2.t.sol

```diff
@@ -128,10 +127,25 @@ contract MOVETokenV2Test is Test {
 
         vm.expectRevert(
             abi.encodeWithSelector(
-                IAccessControl.AccessControlUnauthorizedAccount.selector, multisig, token.MINTER_ROLE()
+                IAccessControl.AccessControlUnauthorizedAccount.selector, multisig, MINTER_ROLE
             )
         );
         token.mint(address(0x1337), 100);
         vm.stopPrank();
     }
+
+    function testCannotGrantRoleFuzz(address messenger, address receiver) public {
```

Comment:
Why is this called a fuzz? Where are you randomizing?

Reply:
foundry has a module that picks any test that starts with test and ends with fuzz. address messenger and address receiver are randomized and ran over and over. This is to assure that AccessControl behaves as expectedly and does not authorize anyone besides the initial DEFAULT_ADMIN_ROLE